PYXAID2/runMD2.py

```diff
@@ -735,114 +726,114 @@ def runMD(params):
                                 print "Time to push matrices is ", tim.stop()
 
 
-                elif info["nspin"]==4:
-                    if info["nk"]==1: # Only one k-point
+                elif info0["nspin"]==2:
 
-                        S = coeff_curr[0].H() * coeff_curr[0]
-                        St = coeff_curr[0].H() * coeff_next[0]  # overlap of wfc at different times
+                    if info0["nk"] == 2:  #single k-point! 
 
+                        # assume we use only the alpha coefficient, similar as PYXAID1, ham() function
+                        # H_dia =  Eii - i*hbar*(<i(t)|j(t+dt)> - <i(t+dt)|j(t)>)
 
-                        sx = S.num_of_cols
```

Comment:
oh, I see - it is still there - just appears as if you deleted it and pasted somewhere else.. hmm..